skytap/resource_skytap_vm.go

```diff
@@ -420,6 +437,54 @@ func updateVMResource(d *schema.ResourceData, meta interface{}, forceRunning boo
 	return resourceSkytapVMRead(d, meta)
 }
 
+func updateVMResourceOptions(d *schema.ResourceData, client *skytap.VMsService, ctx *context.Context, environmentID string, id string) error {
```

Comment:
Rename to  configureVMResource, add documentation a comment why the configuration is necessary after creation,

Reply:
This recommendation was done in PR #13. The create and update flow have been clearly delineated.